iothub/service/src/Common/Security/CryptoKeyGenerator.cs

```diff
@@ -22,8 +25,8 @@ namespace Microsoft.Azure.Devices.Common
     static public class CryptoKeyGenerator
```

Comment:
The order of the modifiers here is throwing me off. Can you change this to "public static class" instead?